templates/src/middleware/auth/index.js

```diff
@@ -4,22 +4,34 @@ const authMiddleware = (req, res, next) => {
    *  1. X-User-Id
    *  2. X-User-Email
    * */
-  const hasUserData = req.headers["x-user-id"] && req.headers["x-user-email"];
-  if (!hasUserData) {
-    res.status(401);
-    res.json({
-      success: false,
-      message: "unauthenticated",
-    });
-  } else {
-    req.user = {
-      id: req.headers["x-user-id"],
-      email: req.headers["x-user-email"],
-    };
+  if (inWhitelist(req.path)) {
     next();
+  } else {
+    const hasUserData = req.headers["x-user-id"] && req.headers["x-user-email"];
+    if (!hasUserData) {
+      res.status(401);
+      res.json({
+        success: false,
+        message: "unauthenticated",
+      });
+    } else {
+      req.user = {
+        id: req.headers["x-user-id"],
+        email: req.headers["x-user-email"],
+      };
+      next();
+    }
   }
+
 };
 
+//whitelist defines the paths that will not pass the authentication middleware
+const auth_whitelist = ["/status/ready", "/status/alive", "/status/about"];
```

Comment:
The other variables are camel case. Also if you're changing it, mind calling it something like `authAllowlist`?

Reply:
Renamed it to authAllowlist. Thanks.